### PR TITLE
Add cloudflare

### PR DESCRIPTION
**Pull request by Isengo1989**
Add cloudflare template info and readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +77,14 @@ An image cache & resize service. Manipulate images on-the-fly with a worldwide c
 Template example: `https://token.cloudimg.io/v7/{mediaUrl}/{mediaPath}&w={width}&h={height}`  
 will become `https://token.cloudimg.io/v7/https://www.example.com/media/01/82/69/sasse.png&w=800&h=800`
 
+### cloudflare [Link](https://www.cloudflare.com)
+
+`Has a lossless image optimization and big hosters have their NFS connected in their contracts directly to cloudflare`  
+An image cache & resize service. Manipulate images on-the-fly with a worldwide cache.  
+Template example: `{mediaUrl}/cdn-cgi/image/width={width},height={height}quality=75/{mediaPath}`  
+will become `https://yourshop.com/cdn-cgi/image/width%3D3000%2Cheight%3D3000quality%3D75/media/64/fd/98/1624446656/ff310899329c4d94855a096a26b02d5e.jpg`
```
Comment (tinect) [inline README.md:85]:
```suggestion
Template example: `{mediaUrl}/cdn-cgi/image/width={width},height={height},quality=75/{mediaPath}`  
will become `https://yourshop.com/cdn-cgi/image/width%3D3000%2Cheight%3D3000%2Cquality%3D75/media/64/fd/98/1624446656/ff310899329c4d94855a096a26b02d5e.jpg`
```

File: README.md [hunk 2]
```diff
@@ -77,6 +77,14 @@ An image cache & resize service. Manipulate images on-the-fly with a worldwide c
 Template example: `https://token.cloudimg.io/v7/{mediaUrl}/{mediaPath}&w={width}&h={height}`  
 will become `https://token.cloudimg.io/v7/https://www.example.com/media/01/82/69/sasse.png&w=800&h=800`
 
+### cloudflare [Link](https://www.cloudflare.com)
+
+`Has a lossless image optimization and big hosters have their NFS connected in their contracts directly to cloudflare`
```
Comment (tinect) [inline README.md:82]:
```suggestion
`Has a lossless image optimization and big hosters have their NFS connected in their contracts directly to cloudflare. Available in Business and Enterprise Plans.`  
```

File: README.md [hunk 3]
```diff
@@ -77,6 +77,14 @@ An image cache & resize service. Manipulate images on-the-fly with a worldwide c
 Template example: `https://token.cloudimg.io/v7/{mediaUrl}/{mediaPath}&w={width}&h={height}`  
 will become `https://token.cloudimg.io/v7/https://www.example.com/media/01/82/69/sasse.png&w=800&h=800`
 
+### cloudflare [Link](https://www.cloudflare.com)
```
Comment (tinect) [inline README.md:80]:
```suggestion
### Cloudflare Image Resizing [Link](https://developers.cloudflare.com/images/)
```

File: src/Resources/app/administration/src/frosh-thumbnail-processor-info-texts.html.twig [hunk 4]
```diff
@@ -46,5 +46,15 @@
...
+
+        <h4 class="supports-headline"><a href="https://www.cloudflare.com" target="_blank">cloudflare</a></h4>
+        <blockquote>Opinion: fast and including webp</blockquote>
+        <p class="supports-description">
+            Has a lossless image optimization and big hosters have their NFS connected in their contracts directly to cloudflare<br><br>
+            <sw-text-field label="Template example" size="small" :copyable="true"
+                           value="{mediaUrl}/cdn-cgi/image/width={width},height={height}quality=75/{mediaPath}"/>
+            will become
+            <code>https://yourshop.com/cdn-cgi/image/width%3D3000%2Cheight%3D3000quality%3D75/media/64/fd/98/1624446656/ff310899329c4d94855a096a26b02d5e.jpg</code>
+        </p>
```
Comment (tinect) [inline src/Resources/app/administration/src/frosh-thumbnail-processor-info-texts.html.twig:58]:
```suggestion
        <h4 class="supports-headline"><a href="https://www.cloudflare.com" target="_blank">cloudflare</a></h4>
        <blockquote>Opinion: fast and including webp</blockquote>
        <p class="supports-description">
            Has a lossless image optimization and big hosters have their NFS connected in their contracts directly to cloudflare. Available in Business and Enterprise Plans.<br><br>
            <sw-text-field label="Template example" size="small" :copyable="true"
                           value="{mediaUrl}/cdn-cgi/image/width={width},height={height},quality=75/{mediaPath}"/>
            will become
            <code>https://yourshop.com/cdn-cgi/image/width%3D3000%2Cheight%3D3000%2Cquality%3D75/media/64/fd/98/1624446656/ff310899329c4d94855a096a26b02d5e.jpg</code>
        </p>
```

**Review by tinect — CHANGES_REQUESTED** [review]

**Comment by Isengo1989** [comment]
@tinect there was an overlap - alright like that?

File: README.md [hunk 5]
```diff
@@ -77,6 +77,14 @@ An image cache & resize service. Manipulate images on-the-fly with a worldwide c
 Template example: `https://token.cloudimg.io/v7/{mediaUrl}/{mediaPath}&w={width}&h={height}`  
 will become `https://token.cloudimg.io/v7/https://www.example.com/media/01/82/69/sasse.png&w=800&h=800`
 
+### Cloudflare Image Resizing [Link](https://developers.cloudflare.com/images/)
+
+`Has a lossless image optimization and big hosters have their NFS connected in their contracts directly to cloudflare (only for business and enterprise)`  
+An image cache & resize service. Manipulate images on-the-fly with a worldwide cache.  
+Template example: `{mediaUrl}/cdn-cgi/image/width={width},height={height},quality=85,format=webp/{mediaPath}`  
+will become `https://yourshop.com/cdn-cgi/image/width%3D3000%2Cheight%3D3000quality%3D85/media/64/fd/98/1624446656/ff310899329c4d94855a096a26b02d5e.jpg`
```
Comment (tinect) [inline README.md:85]: The example result link isn't correct. The comma and format is missing.

File: src/Resources/app/administration/src/frosh-thumbnail-processor-info-texts.html.twig [hunk 6]
```diff
@@ -46,5 +46,15 @@
...
         </p>
+
+        <h4 class="supports-headline"><a href="https://www.cloudflare.com" target="_blank">cloudflare</a></h4>
+        <blockquote>Opinion: many options but only for business and enterprise customers</blockquote>
+        <p class="supports-description">
+            Has a lossless image optimization and big hosters have their NFS connected in their contracts directly to cloudflare<br><br>
+            <sw-text-field label="Template example" size="small" :copyable="true"
+                           value="{mediaUrl}/cdn-cgi/image/width={width},height={height},quality=85,format=webp/{mediaPath}"/>
+            will become
+            <code>https://yourshop.com/cdn-cgi/image/width%3D3000%2Cheight%3D3000quality%3D85/media/64/fd/98/1624446656/ff310899329c4d94855a096a26b02d5e.jpg</code>
```
Comment (tinect) [inline src/Resources/app/administration/src/frosh-thumbnail-processor-info-texts.html.twig:57]: The example result link isn't correct. The comma and format is missing.

**Review by tinect — CHANGES_REQUESTED** [review]

File: src/Resources/app/administration/src/frosh-thumbnail-processor-info-texts.html.twig [hunk 7]
```diff
@@ -46,5 +46,15 @@
             will become
             <code>https://token.cloudimg.io/v7/https://www.example.com/media/01/82/69/sasse.png&w=800&h=800</code>
         </p>
+
+        <h4 class="supports-headline"><a href="https://www.cloudflare.com" target="_blank">cloudflare</a></h4>
+        <blockquote>Opinion: many options but only for business and enterprise customers</blockquote>
```
Comment (tinect) [inline src/Resources/app/administration/src/frosh-thumbnail-processor-info-texts.html.twig:51]:
```suggestion
        <blockquote>Opinion: many options but only for business and enterprise customers. Has no auto detection of supported formats of visitor. So you are pinned to jpeg, webp or avif.</blockquote>
```

File: README.md [hunk 8]
```diff
@@ -77,6 +77,14 @@ An image cache & resize service. Manipulate images on-the-fly with a worldwide c
 Template example: `https://token.cloudimg.io/v7/{mediaUrl}/{mediaPath}&w={width}&h={height}`  
 will become `https://token.cloudimg.io/v7/https://www.example.com/media/01/82/69/sasse.png&w=800&h=800`
 
+### Cloudflare Image Resizing [Link](https://developers.cloudflare.com/images/)
+
+`Has a lossless image optimization and big hosters have their NFS connected in their contracts directly to cloudflare (only for business and enterprise)`  
+An image cache & resize service. Manipulate images on-the-fly with a worldwide cache.
```
Comment (tinect) [inline README.md:83]:
```suggestion
`Opinion: many options but only for business and enterprise customers. Has no auto detection of supported formats of visitor. So you are pinned to jpeg, webp or avif.`  
Has a lossless image optimization and big hosters have their NFS connected in their contracts directly to cloudflare 
```

**Review by tinect — CHANGES_REQUESTED** [review]

**Comment by Isengo1989** [comment]
Resolved the issues before with comma already in a change and commited your changes regarding auto webp switch

**Comment by Isengo1989** [comment]
@tinect I did find the auto option, so now IE and Old Safaris work too, also updated the encoded URL.

**Pull request merged**